src/Components/ArticleList.jsx

```diff
@@ -4,15 +4,15 @@ import { Link } from "react-router-dom";
 import ArticleCard from "./ArticleCard";
 import "./ArticleList.css";
 
-export default function ArticleList() {
+export default function ArticleList({ topic }) {
```

Comment:
As a small note, it's very difficult to read the header (where it says 'Welcome to Northcoder News'), reconsider your styling here 🎨 
<img width="1155" alt="image" src="https://github.com/BTCristian/fe-nc-news/assets/151947206/50d764db-b393-4e4a-bdad-e923c6adaf49">